tests/sitenotice_test.php

```diff
@@ -244,19 +246,19 @@ public function test_user_hlink_interact($formdata) {
 
         $user1 = $this->getDataGenerator()->create_user();
         $this->setUser($user1);
-        $allnotices = helper::retrieve_enabled_notices();
+        $allnotices = sitenotice::get_enabled_notices();
         $this->assertEquals(4, count($allnotices));
         $notice1 = array_shift($allnotices);
 
-        $links = helper::retrieve_notice_links($notice1->id);
+        $links = \local_sitenotice\persistent\noticelink::get_notice_link_records($notice1->get('id'));
```

Comment:
Should we add two use statements in the begging of the file?
```
use local_sitenotice\persistent\noticelink;
use local_sitenotice\persistent\linkhistory;
```

Reply:
Added those `use` statements and cleaned up those fully qualified calls.